src/css_parser.rs

```diff
@@ -894,6 +928,65 @@ fn parse_color_builtin<'a>(input: &'a str)
     parse_color_no_hash(color)
 }
 
+/// Parse a color of the form 'rgb([0-255], [0-255], [0-255])', without the leading 'rgb(' or
+/// trailing ')'. Alpha defaults to 255.
+fn parse_color_rgb<'a>(input: &'a str)
+-> Result<ColorU, CssColorParseError<'a>>
+{
+    let mut components = input.split(',').map(|c| c.trim());
+    let color = parse_color_rgb_components(&mut components)?;
+    if let Some(arg) = components.next() {
+        return Err(CssColorParseError::ExtraArguments(arg));
+    }
+    Ok(color)
+}
+
+/// Parse a color of the form 'rgba([0-255], [0-255], [0-255], [0.0-1.0])', without the leading
+/// 'rgba(' or trailing ')'.
+fn parse_color_rgba<'a>(input: &'a str)
+-> Result<ColorU, CssColorParseError<'a>>
+{
+    let mut components = input.split(',').map(|c| c.trim());
+    let rgb_color = parse_color_rgb_components(&mut components)?;
+    let a = components.next().ok_or(CssColorParseError::MissingColorComponent("alpha"))?;
+    if a == "" {
```

Comment:
Use `a.is_empty()` or `a.len() == 0` instead of comparing directly to an empty string.